packages/docusaurus-plugin-content-docs/src/docFrontMatter.ts

```diff
@@ -27,6 +27,10 @@ const DocFrontMatterSchema = Joi.object<DocFrontMatter>({
   slug: Joi.string(),
   sidebar_label: Joi.string(),
   sidebar_position: Joi.number(),
+  tags: [Joi.string(), Joi.array().items(Joi.object({
+    label: Joi.string,
+    permalink: Joi.string
+  }))]
```

Comment:
```suggestion
  tags: Joi.array().items(Joi.alternatives().try(
    Joi.string(),
    Joi.object({
      label: Joi.string,
      permalink: Joi.string
    }),
  )),
```

Joi is really a pain, yeah

Reply:
Actually, there's a `FrontMatterTagsSchema` from `utils-validation`. See here: 

https://github.com/facebook/docusaurus/blob/fd3c0c661944e2aa88330f691e248b436b93b7f7/packages/docusaurus-plugin-content-blog/src/blogFrontMatter.ts#L8-L13

https://github.com/facebook/docusaurus/blob/fd3c0c661944e2aa88330f691e248b436b93b7f7/packages/docusaurus-plugin-content-blog/src/blogFrontMatter.ts#L78

Reply:
thank you!